src/app.vue

```diff
@@ -1,16 +1,205 @@
 <template>
-  <div id="app">
-    <router-view/>
-    <footer />
+  <div v-if="loading" class="loading">
+    <img class="logo" src="./assets/LogoWeniAnimada.svg">
+  </div>
+
+  <div v-else class="app">
+    <div>
+      <Sidebar class="sidebar"/>
+    </div>
+    <div class="content">
+      <Navbar class="navbar"/>
+      
+      <router-view class="page"/>
+    </div>
+
+    <right-sidebar ref="right-sidebar"/>
   </div>
 </template>
 
+<script>
+import Sidebar from './components/external/Sidebar.vue'
+import Navbar from './components/external/navbar.vue'
+import RightSidebar from './components/RightSidebar.vue';
+import account from './api/account';
+import axios from 'axios';
+import Oidc from "oidc-client";
+
+export default {
+  components: {
+    Sidebar,
+    Navbar,
+    RightSidebar,
+  },
+
+  data() {
+    return {
+      loading: true,
+      loadedUser: null,
+    };
+  },
+
+  created() {},
+
+  mounted() {
+    this.$root.$on('manage-members', (data) => {
+      this.$refs['right-sidebar'].open('manage members', data);
```

Comment:
Since you removed the space from other uses, you should remove it from here also
It would become manage-members